modules/grpc/bigquery/bigquery-worker.cpp

```diff
@@ -423,8 +423,8 @@ DestinationWorker::create_channel()
 {
   DestinationDriver *owner_ = this->get_owner();
 
-  auto credentials = ::grpc::GoogleDefaultCredentials();
   ::grpc::ChannelArguments args = this->create_channel_args();
+  auto credentials = this->create_credentials();
```

Comment:
We shouldn't change the default here, BigQuery is most likely being used with `GCAM_ADC`.

Reply:
Nice catch!

Added this to the ctor of `bigquery::DestinationDriver`:
```c++
this->credentials_builder.set_mode(GCAM_ADC);
```